lib/code_corps/policy/policy.ex

```diff
@@ -51,6 +51,9 @@ defmodule CodeCorps.Policy do
   defp can?(%User{} = current_user, :delete, %UserSkill{} = user_skill, %{}), do: Policy.UserSkill.delete?(current_user, user_skill)
   defp can?(%User{} = current_user, :create, %UserRole{} = user_role, %{}), do: Policy.UserRole.create?(current_user, user_role)
   defp can?(%User{} = current_user, :delete, %UserRole{} = user_role, %{}), do: Policy.UserRole.delete?(current_user, user_role)
+  defp can?(%User{} = user, :show, %StripeConnectAccount{} = stripe_connect_account, %{} = params), do: Policy.StripeConnectAccount.show?(user, stripe_connect_account)
```

Comment:
`%{} = params` should throw out a compile warning when you run the tests, because `params` is not being used. Is it not happening for you. `%{}` should be correct.